.env.dist

```diff
@@ -18,6 +18,7 @@ VUE_ALLOWED_HOSTS=profile.example.org
 #
 CLOUDFLARE_EMAIL=
 CLOUDFLARE_API_KEY=
+LETS_ENCRYPT_EMAIL=
```

Comment:
Since it's for proxy, maybe it should go near the bottom of this file in the section "Variables for the proxy container".